src/Components/Create.tsx

```diff
@@ -0,0 +1,163 @@
+    ""
+  );
+  const [isPending, setIsPending] = useState(false);
+  const [rating, setRating] = useState(3);
+  const navigate = useNavigate();
+
+  const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    const travel = {
+      title,
+      country,
+      location,
+      dateRange,
+      description,
+      author,
+      travelCompanions,
+      rating
+    };
+    console.log(travel);
+
+    setIsPending(true);
+
+    fetch("http://localhost:8000/travels/", {
+      method: "POST",
+      headers: { "Content-type": "application/json" },
+      body: JSON.stringify(travel),
+    }).then(() => {
+      console.log("new added");
+      setIsPending(false);
+      
+    });
+  };
+
+  const openNotification = () => {
+    notification.open({
+      message: 'Your new journey is submitted!',
+      description:
+        'Now you can see the newly added element on the home page.',
+    });
+  };
+
+  const onFinishFailed = (errorInfo: any) => {
+    console.log('Failed:', errorInfo);
+  };
+
+  return (
+
+    <div className="form-input">
+      <div className="form-input-head">
+        <h2>Fill the form to add new journey!</h2>
+      </div>
+      <Form
+        labelCol={{ span: 4 }}
+        wrapperCol={{ span: 14 }}
+        layout="horizontal"
+        onSubmitCapture={handleSubmit}
+        onFinishFailed={onFinishFailed}
+      >
+        <Form.Item label="Title" rules={[{ required: true, message: 'Please add a title!' }]}>
+          <Input value={title} onChange={(e) => setTitle(e.target.value)} />
+        </Form.Item>
+
+        <Form.Item label="Country">
+          <Input value={country} onChange={(e) => setCountry(e.target.value)} />
+        </Form.Item>
+
+        <Form.Item label="Location">
+          <Input
+            value={location}
+            onChange={(e) => setLocation(e.target.value)}
+          />
+        </Form.Item>
+
+  {/*       <Form.Item label="Date">
+          <Input
+            type="number"
+            value={date}
+            onChange={(e) => setDate(e.target.value)}
+          />
+        </Form.Item> */}
+          <Form.Item label="RangePicker">
+          <RangePicker  onChange={(e) => {console.log(e?.[0]?.toDate())
+            setDateRange({
+                start: e?.[0]?.toDate(),
+                end: e?.[1]?.toDate()
+            })}
+        }/>
+
+       
+        </Form.Item> 
+
+        <Form.Item label="Description">
+          <TextArea
+            rows={4}
+            value={description}
+            onChange={(e) => setDescription(e.target.value)}
+          />
+        </Form.Item>
+
+        <Form.Item label="Author">
+          <Input value={author} onChange={(e) => setAuthor(e.target.value)} />
+        </Form.Item>
+
+        <Form.Item label="Travel companions">
+          <InputNumber
+            value={travelCompanions}
+            onChange={(e) => setTravelCompanions(e)}
+          />
+        </Form.Item>
```

Comment:
Erdemes eszben tartanod majd, hogy amennyiben egy field nem kotelezoen megadando, akkor majd a megjelenitesnel optional chaninget hasznalj, hogy ne is akarja megjeleniteni a nem megadott elemeket, vagy adj meg default erteket olyannal ahol indokolt.